src/documentation/runners/jstorm.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: "JStorm Runner"
+permalink: /documentation/runners/jstorm/
+redirect_from: /learn/runners/jstorm/
```

Comment:
the redirect_from line shouldn't be needed as this page is new

Reply:
Done